tests/taxReturns.test.js

```diff
@@ -497,6 +528,29 @@ test('can retrieve tax returns', async () => {
   expect(mocker.scope.isDone()).toBeTruthy();
 });
 
+test('can retrieve tax returns_with_type', async () => {
```

Comment:
can you please remove the underscores just for consistency?
```suggestion
test('can retrieve tax returns with type', async () => {
```